### ISSUE TITLE
[Bug] iOS "Large Text" feature breaks the login screen (Google login button is not visible)

### ISSUE THREAD

**Issue by testuser51** [cryptee/web-client#224]
**Describe the bug**
There is no button to log in via Google account
![IMG_3313](https://github.com/user-attachments/assets/99768e67-759d-4a73-b983-4e49ca0a74db)
![IMG000](https://github.com/user-attachments/assets/5fbb56d5-b57f-4506-b5c3-a98834a9ae9a)

**To Reproduce**
"Large text" is enabled in the OS settings:
1. Open settings
2. Display
3. Display zoom
4. Larger text
![386941852-56615102-8c7a-43fa-bc75-b580fd156dc8](https://github.com/user-attachments/assets/e34fe4d1-a300-45a4-aada-19d14471c49b)
5. Open PWA (This problem is not always present in the browser, but always in PWA)

**Expected behavior**
The user sees a button to log in to the account via Google
![image](https://github.com/user-attachments/assets/a13159bb-4573-4cfb-98a7-e3b5e7a6f70c)

**System Information (please complete the following information):**
 - Device: iPhone SE
 - OS: iOS 18.1.1 
 - Browser: Safari/PWA
 - Browser Extensions: AdGuard

**Additional context**
1. "Large text" is enabled in the OS settings:
2. The problem does not occur in Firefox, but it does occur in Safari. The problem is also always! present in PWA

**Comment by johnozbay** [comment]
Hi there! 
Thanks a lot for filing this! 🙏🏻

We actually built Cryptee so that if users use the increase-font-size feature, Cryptee still looks and functions correctly. That way on your devices you can use the zoom in shortcuts (like `cmd +` or on phones use the system settings for increased font sizes) and use Cryptee all the same but with bigger texts, icons and buttons etc. 
 
However, it seems that once Large Text is enabled, instead of increasing the font size, iOS actually reduces the overall resolution of your phone's screen. From 375px to 320px. 

What this means is that not only the overall space available shrinks, but it actually doesn't instruct the browser to increase the font-size. 

We're making some changes for the whole app across the board to make sure things look and function correctly now that we know how the Large Text function works. So with the upcoming update you should be able to use everything as expected. 
 
<img width="485" alt="image" src="https://github.com/user-attachments/assets/684b9d67-5954-46f8-9c2b-002810d0b481">

I have to note however that 320x480 is such a small resolution that — it was in fact the resolution on this nokia 6260 phone from 2004: 
![image](https://github.com/user-attachments/assets/068002c5-70dc-408f-913c-dd97926dd8a6)

It's so small by todays standards that you may still experience some issues in odd parts of the app despite our best efforts. 
We usually test things down to 320px as much as we humanly can, but things like larger lists, or some editor features that may require greater width (i.e. tables with more than 2 columns etc) may simply not work. 

---

Long story short, I can confirm that this is now fixed this in the alpha version and the issue is linked to our internal issue/release tracker. Once the update with this fix is out, this issue will be closed automatically, and you'll get a notification!

Thank you so much for filing this, your help and patience!
